package.json

```diff
@@ -12,9 +12,7 @@
     "node": ">=10.13.0"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "5.3.13",
     "@storybook/addon-docs": "5.3.12",
```

Comment:
Should we update @storybook/addon-docs and @storybook/react to the latest version already (5.3.14 both)?